app/styles/ui/dialogs/_rebase-conflicts.scss

```diff
@@ -0,0 +1,124 @@
+@import '../../mixins';
+
+dialog#rebase-conflicts-list {
+  width: 500px;
+
+  .summary {
+    margin-bottom: 20px;
+  }
+
+  .dialog-header h1 {
+    font-weight: var(--font-weight-light);
+  }
+
+  .dialog-content {
+    padding-bottom: var(--spacing);
+  }
+
+  .green-circle {
+    background-color: var(--color-new);
+    color: var(--background-color);
+    border-radius: 50%;
+    height: 22px;
+    width: 22px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+  ul {
+    list-style: none;
+    padding: 0;
+    max-height: 285px;
+    overflow-y: auto;
+
+    li {
+      margin-bottom: var(--spacing-double);
+      padding-left: 0;
+    }
+
+    li:last-of-type {
+      margin-bottom: calc(var(--spacing) * 2);
+    }
+
+    li.unmerged-file-status-resolved,
+    li.unmerged-file-status-conflicts {
+      display: flex;
+      flex-flow: row nowrap;
+
+      .file-octicon {
+        margin-right: var(--spacing);
+      }
+
+      .column-left {
+        display: flex;
+        flex-flow: column nowrap;
+        align-items: start;
+        max-width: 50%;
+        padding-right: var(--spacing);
+
+        .path-text-component {
+          text-overflow: ellipsis;
+          max-width: 100%;
+        }
+      }
+
+      .action-buttons {
+        margin-left: auto;
+        flex-shrink: 0;
+        flex: 0 1 auto;
+        display: flex;
+        align-items: center;
+        justify-content: center;
+      }
+
+      .green-circle:last-child {
+        margin-left: auto;
+        margin-top: var(--spacing);
+        flex-shrink: 0;
+        flex: 0 1 auto;
+      }
+    }
+
+    .unmerged-file-status-resolved .file-conflicts-status {
+      color: $green;
+    }
+
+    .unmerged-file-status-conflicts {
+      .file-conflicts-status {
+        color: $orange;
+      }
+
+      .command-line-hint {
+        color: $gray;
+      }
+    }
+  }
+
+  .all-conflicts-resolved {
+    display: flex;
+    flex-flow: row nowrap;
+    padding: var(--spacing) 0 var(--spacing-double);
+
+    .message {
+      padding-left: var(--spacing);
+      padding-top: var(--spacing-third);
+    }
+  }
+
+  .cli-link {
```

Comment:
I don't see this class being used anywhere?

Reply:
Cleaned up in ac9ddb4b4500c9b28f7e582497ba7d76a6029ecc